composer.json

```diff
@@ -14,13 +14,13 @@
   "require-dev": {
     "ec-europa/oe-code-review": "^0.1",
     "ec-europa/oe-poetry-behat": "^0.3",
-    "ec-europa/oe-task-runner": "^0.3",
+    "ec-europa/oe-task-runner": "dev-OPENEUROPA-206",
```

Comment:
I understand this will change once the task runner PR is merged but so my question is: at which stage of the ticket do we update this line?

Reply:
Yes correct, as soon as review is done I'll first close the issue on the task runner and then I'll change this, have the test pass and merge.